### PR TITLE
Shipping Labels: substituted `VStack` with `LazyVStack` in SwiftUI views

### PR DESCRIPTION
**Pull request by pmusolino**
Since we dropped iOS 13, now we can use `LazyVStack` instead of `VStack` on Shipping Labels views implemented in SwiftUI. The three standard stack views, `HStack`, `VStack`, and `ZStack`, all load their contained view hierarchy when they display, and loading large numbers of views all at once can result in slow runtime performance. Using `LazyVStack` the stack is “lazy,” in that the stack view doesn’t create items until it needs to render them onscreen.

I also updated the method `navigationBarTitle` with `navigationTitle` available on iOS 14.

## Testing
Just check that everything in the Shipping Label flow works like before.

Update release notes:
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift [hunk 1]
```diff
@@ -42,7 +41,7 @@ struct ShippingLabelPackageList: View {
             .background(Color(.systemBackground))
         }
         .background(Color(.listBackground))
-        .navigationBarTitle(Text(Localization.title), displayMode: .inline)
+        .navigationTitle(Localization.title)
```
Comment (Ecarrion) [inline WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift:44]: [Q] Why this change by mistake? 😅

**Review by Ecarrion — APPROVED** [review]
LGTM!

File: WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift (hunk 1 above)
Comment (pmusolino) [reply WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift:44]: Because of this `I also updated the method navigationBarTitle with navigationTitle available on iOS 14.`. It's just a new way to declare the bar title in iOS 14. 👍
